trompace/mutations/property.py

```diff
@@ -0,0 +1,101 @@
+# Generate GraphQL queries for mutations pertaining to software applications.
+
+from .templates import mutation_create, mutation_update, mutation_delete, mutation_link
+from . import StringConstant, bool_to_str
+
+
+CREATE_PROPERTY = '''
+CreateProperty(
+{parameters}
+) {{
+  identifier
+}}
+'''
+
+CREATE_PROPERTYVALUESPECIFICATION = '''
+CreatePropertyValueSpecification(
+{parameters}
+) {{
+  identifier
+}}
+'''
+
+ADD_CONTROLACTION_PROPERTYVALUESPECIFICATION  = '''
+AddActionInterfaceThingInterfacee(
+  from: {{identifier: "{identifier_1}"}}
+  to: {{identifier: "{identifier_2}"}}
+  field: object
+)
+{{
+    from {{
+        --typename
```

Comment:
This is what it says in the CE API manual.

Reply:
I don't recognise the `--typename` syntax as valid graphql. Are you sure it's not supposed to be `__typename` instead? If it's using hyphens in the manual then this needs to be updated. Can you make an annotation in the manual so that we remember to fix it